xmind/tests/test_workbook_element.py

```diff
@@ -0,0 +1,751 @@
+import logging
+from . import base
+from unittest.mock import patch, MagicMock, call, Mock
+from xmind.core.workbook import WorkbookElement
+from xmind.core.const import TAG_WORKBOOK, NAMESPACE, XMAP, NS_FO, NS_XHTML, NS_XLINK, NS_SVG, TAG_SHEET, ATTR_VERSION
+
+
+class TestWorkbookElement(base.Base):
+
+    def setUp(self):
+        super(TestWorkbookElement, self).setUp()
+
+        self._workbook_mixin_element_init = self._init_patch_with_name(
```

Comment:
I think it will work, but originally idea is to mock class/method based on its import
in workbook.py we have next import 
```from .mixin import WorkbookMixinElement```
and later in ```__init__```

> def ```__init__```(self, node=None, ownerWorkbook=None):
>        super(WorkbookElement, self).```__init__```(node, ownerWorkbook)

and then your test imports 
```from xmind.core.workbook import WorkbookElement```
that means that you have to mock path as it comes to your test and WorkbookMixinElement in this case comes from ```xmind.core.workbook```

> ```'xmind.core.**workbook**.WorkbookMixinElement.__init__'```
instead of 
> ```'xmind.core.**topic**.WorkbookMixinElement.__init__'```

**Additional one point to do it in the way I ask you** is
if tomorrow we will do a refactoring of methods/classes in core and we decided to move classes to subfolder then your tests will start to fail because of hardcoded original place of file in module.